spec/factories/filesystem.rb

```diff
@@ -36,4 +36,189 @@
   factory :filesystem_txt_file, :parent => :filesystem do
     name "periodical/utilities/blue_screen_description.txt"
   end
+
+  #######################################
+  ##### Variations of nice format #######
```

Comment:
Yes; this arrangement makes the factories look more like fixtures.

A factory is about filling in the "baseline" attributes: ones that matter, should be set in the spec, right next to the relevant assertions.